src/Storages/Distributed/DirectoryMonitor.cpp

```diff
@@ -854,6 +870,10 @@ struct StorageDistributedDirectoryMonitor::Batch
             ReadBufferFromFile in(file_path->second);
             const auto & distributed_header = readDistributedHeader(in, parent.log);
 
+            OpenTelemetryThreadTraceContextScope thread_trace_context(__PRETTY_FUNCTION__,
```

Comment:
(Cosmetics again (sorry): let's do the formatting like that:
```
OpenTelemetryThreadTraceContextScope thread_trace_context(__PRETTY_FUNCTION__,
    distributed_header.client_info.client_trace_context,
    parent.storage.getContext()->getOpenTelemetrySpanLog());
```
e.g. see l. 867 for a related example)